homeassistant/components/mobile_app/http_api.py

```diff
@@ -91,8 +91,9 @@ async def post(self, request: Request, data: dict) -> Response:
         )
 
         remote_ui_url = None
-        with suppress(hass.components.cloud.CloudNotAvailable):
-            remote_ui_url = cloud.async_remote_ui_url(hass)
+        if cloud.async_active_subscription(hass):
+            with suppress(hass.components.cloud.CloudNotAvailable):
```

Comment:
I guess we don’t need this check anymore?

Reply:
It can still be unavailable, for example if cloud was turned off manually.